dhis-2/dhis-services/dhis-service-core/src/main/java/org/hisp/dhis/security/TwoFactorUtils.java

```diff
@@ -34,12 +34,13 @@
 import org.apache.commons.lang3.StringUtils;
 import org.hisp.dhis.user.User;
 import org.jboss.aerogear.security.otp.Totp;
-import org.springframework.util.Assert;
+
+import com.google.common.base.Strings;
 
 /**
  * @author Henning Håkonsen
  */
-public class SecurityUtils
+public class TwoFactorUtils
```

Comment:
Rename to `TwoFactoryAuthenticationUtils` to make it more descriptive (TwoFactor does not imply security or auth).